src/ResourceManager/ApiManagement/AzureRM.ApiManagement.psd1

```diff
@@ -140,7 +140,11 @@ CmdletsToExport = 'Add-AzureRmApiManagementRegion',
                'Set-AzureRmApiManagementProperty', 
                'Set-AzureRmApiManagementSubscription', 
                'Set-AzureRmApiManagementUser', 
-               'Set-AzureRmApiManagementTenantAccess'
+               'Set-AzureRmApiManagementTenantAccess',
```

Comment:
Please summarize these changes in the 'Current Release' section of your changelog document: https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/ApiManagement/ChangeLog.md